sanic/config.py

```diff
@@ -1,8 +1,19 @@
-import os
-import types
+from os import environ as os_environ
+
+from typing import Union, \
+                   Any
 
+from .utils import str_to_bool, \
+                   load_module_from_file_location
+
+# \/ \/ \/ \/
```

Comment:
Can It be like that

Reply:
Like what? I can't understand, be honest.